recommender/engine/rl_agent/models/actor_model.py

```diff
@@ -29,9 +62,12 @@ def forward(self,
 
         """
 
-        services_history = self.history_embedder(services_history)
+        embedded_services_history = self.history_embedder(services_history)
+        x = torch.cat([embedded_services_history, user, filters, search_phrase], dim=1)
+
+        for layer in self.layers:
+            x = layer(x)
+            x = F.relu(x)
```

Comment:
You can use [Sequential](ActorModel(torch.nn.Module)) - it will automatically process `for loop` for you - without explicitly coding it.